api.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python3
+import argparse
+import os
+import logging
+import tornado
+import signal
+
+from logging.handlers import TimedRotatingFileHandler
+from pymongo import MongoClient
+from queue import Queue
+
+import config
+os.makedirs(config.LOGS_DIR, exist_ok=True)
+logging.basicConfig(
+    handlers=[
+        TimedRotatingFileHandler(
+            "{}/log".format(config.LOGS_DIR), 
+            when=config.LOGS_ROTATE_WHEN, 
+            backupCount=config.LOGS_BACKUP_COUNT
+        ),
+        logging.StreamHandler()
+    ],
+    level=logging.INFO if not config.DEBUG else logging.DEBUG
+)
+
+from broadway_api.callbacks import *
+from broadway_api.handlers.client import *
+from broadway_api.handlers.worker import *
+from broadway_api.utils.bootstrap import *
```

Comment:
`import *` are usually code smells because they overwrite objects in different namespaces (could we also move the imports up to the top of the file).

Reply:
This will be fixed upstream (linter will throw warnings about it anyways).